inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/exec_graph_serialization.cpp

```diff
@@ -139,23 +139,23 @@ const char expected_serialized_model[] = R"V0G0N(
 <net name="addmul_abc" version="10">
 	<layers>
 		<layer id="0" name="C" type="Input">
-			<data execOrder="3" execTimeMcs="not_executed" originalLayersNames="C" outputLayouts="x" outputPrecisions="FP32" primitiveType="unknown_FP32" runtimePrecision="FP32" />
+			<data shape="1" element_type="f32" execOrder="3" execTimeMcs="not_executed" originalLayersNames="C" outputLayouts="x" outputPrecisions="FP32" primitiveType="unknown_FP32" runtimePrecision="FP32" />
 			<output>
 				<port id="0" precision="FP32">
 					<dim>1</dim>
 				</port>
 			</output>
 		</layer>
 		<layer id="1" name="B" type="Input">
-			<data execOrder="1" execTimeMcs="not_executed" originalLayersNames="B" outputLayouts="x" outputPrecisions="FP32" primitiveType="unknown_FP32" runtimePrecision="FP32"/>
+			<data shape="1" element_type="f32" execOrder="1" execTimeMcs="not_executed" originalLayersNames="B" outputLayouts="x" outputPrecisions="FP32" primitiveType="unknown_FP32" runtimePrecision="FP32"/>
```

Comment:
Do we need to have element type and shape here?

Reply:
Yes, just because it is regular operations serialization. 
Actually data field should be used for operations attributes and nodes in execution graph may have them. But attributes in rt_info must be located in a separate section but until we don't have it we need to keep rt attributes inside data field.

Reply:
BTW now plugins can add operation attributes inside FrameworkNode so they also will be serialized into data field.